### PR TITLE
[refactor] method improvement rationale

### PR DESCRIPTION
**Pull request by dukbong**
## What's changed?

The improved method below utilizes the StringUtils.isAnyEmpty() method from the Apache Commons Lang library to enhance readability and maintainability. This change improves code conciseness and clarity while reducing repetitive null and empty string checks. Moreover, using these utility methods makes the code's intent clearer and decreases the likelihood of errors.
## Checklist

- [ ]  I have read the [Contributing Guide](https://hertzbeat.apache.org/docs/community/code_style_and_quality_guide)
- [ ]  I have written the necessary doc or comment.
- [ ]  I have added the necessary unit tests and all cases have passed.

## Add or update API

- [ ] I have added the necessary [e2e tests](../e2e) and all cases have passed.

### CODE REVIEW

File: collector/src/main/java/org/apache/hertzbeat/collector/util/CollectUtil.java [hunk 1]
```diff
@@ -309,7 +311,7 @@ public static JsonElement replaceSmilingPlaceholder(JsonElement jsonElement, Map
                         Map<String, String> map = JsonUtil.fromJson(jsonValue, typeReference);
                         if (map != null) {
                             map.forEach((name, value) -> {
-                                if (name != null && !"".equals(name.trim())) {
+                                if (StringUtils.isEmpty(name)) {
```
Comment (tomsun28) [inline collector/src/main/java/org/apache/hertzbeat/collector/util/CollectUtil.java:314]: hi, here is not empty name

**Review by tomsun28 — COMMENTED** [review]
👍👍

File: collector/src/main/java/org/apache/hertzbeat/collector/util/CollectUtil.java (hunk 1 above)
Comment (dukbong) [reply collector/src/main/java/org/apache/hertzbeat/collector/util/CollectUtil.java]: Thanks! @tomsun28 
I have fixed it!!! 👍

**Review by TJxiaobao — APPROVED** [review]
👍👍👍！！！

**Comment by dukbong** [comment]
Thanks!! @TJxiaobao !!

**Pull request merged**